### PR TITLE
Use deck.gl mesh layer

### PR DESCRIPTION
**Pull request by xintongxia**

### CODE REVIEW

File: modules/layers/test/lib/mode-handlers/draw-line-string-handler.test.js [hunk 1]
```diff
@@ -81,8 +81,7 @@ describe('when no selection', () => {
       },
       featureIndexes: [featureCollection.features.length],
       editContext: {
-      featureIndexes: [featureCollection.features.length],
-
+        featureIndexes: [featureCollection.features.length]
```
Comment (georgios-uber) [inline modules/layers/test/lib/mode-handlers/draw-line-string-handler.test.js]: unrelated

Comment (xintongxia) [reply modules/layers/test/lib/mode-handlers/draw-line-string-handler.test.js]: yarn pretty did

**Comment by xintongxia** [comment]
> Did you test the segment layer? Are the directional arrows showing?

@georgios-uber No, I don't have much context about the impact of this change. Is there something else I need verify?

**Comment by georgios-uber** [comment]
> > Did you test the segment layer? Are the directional arrows showing?
> 
> @georgios-uber No, I don't have much context about the impact of this change. Is there something else I need verify?

try https://github.com/uber/nebula.gl/tree/master/examples/sf running this

**Comment by georgios-uber** [comment]
Taking this over.
